docs/source/user_guide/tutorials/getting_started.rst

```diff
@@ -4,733 +4,691 @@
+================================
+Next, let's modify our naive pipeline configuration YAML to try to improve our results. Primarily, we 
+will change the ``COMPARISONS`` we pass to ``splink_evaluating_pairs`` to use flexible comparison 
+methods rather than exact matches, allowing us to link records which have typos or other noise in them. We'll 
+use a new pipeline configuration YAML, :download:`pipeline_demo_improved.yaml`, with these changes.
+
+In ``splink_evaluating_pairs``, we make the following change:
+
+.. code-block:: diff
 
-   [150000 rows x 8 columns]
+     LINK_ONLY: true
+     BLOCKING_RULES_FOR_TRAINING: "'l.first_name == r.first_name,l.last_name == r.last_name'"
+  -  COMPARISONS: "ssn:exact,first_name:exact,middle_initial:exact,last_name:exact"
+  +  COMPARISONS: "ssn:levenshtein,first_name:name,middle_initial:exact,last_name:name"
+     PROBABILITY_TWO_RANDOM_RECORDS_MATCH: 0.0001  # == 1 / len(w2)
+
+
+
+``COMPARISONS`` now uses 
+`Levenshtein <https://moj-analytical-services.github.io/splink/api_docs/comparison_library.html#splink.comparison_library.LevenshteinAtThresholds>`_
+comparisons for ``ssn``, and 
+`Name <https://moj-analytical-services.github.io/splink/api_docs/comparison_library.html#splink.comparison_library.NameComparison>`_
+comparisons for ``first_name`` and ``last_name``, to link similar but not identical SSNs and names.
+
+By re-running the pipeline with these changes and then running the evauation script, we can see how our results compare::
+
+  $ easylink run -p pipeline_demo_improved.yaml -i input_data_demo.yaml -e environment_local.yaml -I /mnt/team/simulation_science/priv/engineering/er_ecosystem/images
+  $ python print_fp_fn_w2_ssa.py results/2025_06_26_11_08_57 .996
+  9292 true links
+  For threshold 0.996, len(false_positives)=19; len(false_negatives)=158
+
+We eliminated 30 false negatives compared to the naive results, thanks to our model linking more records with columns that 
+are similar but don't exactly match.
+
+Linking 2030 datasets using improved pipeline
+=============================================
+Finally, let's run this same "improved" pipeline, but using :download:`input_data_demo_2030.yaml` 
+as the input YAML, which uses the SSA and W-2 datasets from 2030 rather than 
+2020. Like we did for 2020, we'll create a ``2030`` directory and save :download:`input_file_ssa.parquet <2030/input_file_ssa.parquet>` and 
+:download:`input_file_w2.parquet <2030/input_file_w2.parquet>` into it.
+
+We can run the same pipeline on different data by changing only the input parameter::
+
+  $ easylink run -p pipeline_demo_improved.yaml -i input_data_demo_2030.yaml -e environment_local.yaml -I /mnt/team/simulation_science/priv/engineering/er_ecosystem/images
+  python print_fp_fn_w2_ssa.py results/2025_06_26_11_17_52 .996
+  10345 true links
+  For threshold 0.996, len(false_positives)=14; len(false_negatives)=149
+
+We get similar, but not identical, results with the 2030 data.
+
+
+.. note::
```

Comment:
Change to a "Conclusion" (or more fun name) heading

"In this tutorial, we've demonstrated..."

"Not everything EasyLink can do has been covered in this tutorial. EasyLink currently includes a few more implementations we haven't used here, can run pipelines on a computational cluster managed by Slurm [link] or distribute work using Apache Spark [link], and has additional flexibility in the pipeline schema that we haven't demonstrated."